blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_7_serviceability_limit_state.py

```diff
@@ -0,0 +1,49 @@
+"""Module including all formulas from chapter 7 of NEN-EN 1992-1-1+C2:2011."""
+# pylint: disable=arguments-differ
+from blueprints.codes.formula import Formula
+from blueprints.typehints import KN, KN_TO_N, MM2, MPA
+
+
+class Form7Dot3CoefficientKc(Formula):
+    """Class representing the formula 7.3 for the coefficient kc for flanges of tubular cross-sections and T-sections [-]."""
+
+    label = "7.3"
+    source_document = "NEN-EN 1992-1-1+C2:2011"
+
+    def __init__(
+        self,
+        f_cr: KN,
+        a_ct: MM2,
+        f_ct_eff: MPA,
```

Comment:
I think we have to settle what units we are using over the complete project--> if we want the formulas to be able to be inside a pipeline we shouldn't have to worry about units. So we can do a few things: 1. agree on standard units in the library and use type aliases 2. use a library like pint for the unit conversions 3. do unit checks ourselves and extend our in and output types with a unit.

I know it is some work, but if we don't do this correctly I think we will have a lot of headache in the future. 

Maybe we should start a private discussion on this?

I would actually suggest to do Newton as input instead of KN --> less unit conversions and it is equal to the input into software like say: idea and scia

Reply:
I agree on the private discussion. I'll open one

Reply:
In our last maintainers meeting, we agreed to avoid standard units across the package, opting instead for units that are most commonly used in specific formulas. For instance, we discussed using kNm for bending moments as it's more conventional than Nm or Nmm. Similarly, mm² was preferred over m² for cross-sectional areas. I welcome further discussion on this topic and agree on the importance of getting it right. One essential point I believe we should adhere to is clearly stating the required units in both the type hints and docstrings of each Formula.
I will start a private discussion on this matter.